charts/argo-cd/templates/_common.tpl

```diff
@@ -79,21 +79,15 @@ Node affinity
 {{- else -}}
 {{- $preset := .context.Values.global.affinity -}}
 podAntiAffinity:
-  preferredDuringSchedulingIgnoredDuringExecution:
   {{- if (eq $preset.podAntiAffinity "soft") }}
+  preferredDuringSchedulingIgnoredDuringExecution:
   - weight: 100
     podAffinityTerm:
       labelSelector:
         matchLabels:
           app.kubernetes.io/name: {{ include "argo-cd.name" .context }}-{{ .component.name }}
       topologyKey: kubernetes.io/hostname
   {{- else }}
-  - weight: 100
```

Comment:
Why is preference for zone removed from hard affinity?

Reply:
This is taken from HA setup here: https://github.com/argoproj/argo-cd/blob/ba86be2fcbab6c53c311426a31d9761ff6934300/manifests/ha/base/overlays/argocd-repo-server-deployment.yaml#L10

Reply:
Dear @pdrastil 
I am truly sorry to bother you. It's just a mistake from me. I will close this PR.
Best regards.

Reply:
Hi @laiminhtrung1997 - no problem at all.